tendrl/common/flows/flow.py

```diff
@@ -0,0 +1,96 @@
+import etcd
+import logging
+
+from tendrl.common.flows.base_flow import BaseFlow
+from tendrl.common.flows import utils
+
+LOG = logging.getLogger(__name__)
+
+
+class Flow(BaseFlow):
```

Comment:
If we remove flow.py and move everything in base_flow.py, this class already has got a run() function and there is no way we can impose all subclasses to implement run() function. Is there any other way to achieve this?